test/test_p4est_2d.jl

```diff
@@ -23,6 +23,14 @@ isdir(outdir) && rm(outdir, recursive=true)
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_nonconforming_flag.jl"),
       l2   = [3.198940059144588e-5],
       linf = [0.00030636069494005547])
+
+    # Check for allocations (type instabilities) coming from mortars
```

Comment:
```suggestion
    # Ensure that we do not have excessive memory allocations (e.g., from type instabilities)
```
I'd reformulate this comment (here and in the other places) a little more generic, since IMHO it is generally a good check